csfieldguide/templates/chapters/chapter.html

```diff
@@ -12,26 +12,37 @@
 {% endblock page_heading %}
 
 {% block content %}
-  {% if chapter.video %}
-    <div class="row mb-3">
-      <div class="col-sm-12 col-md-10 offset-md-1 text-center">
-        <div class="embed-responsive embed-responsive-16by9">
-          <iframe src="{{ chapter.video }}" class="embed-responsive-item" frameborder="0" allowfullscreen></iframe>
+  {% if chapter.translation_available %}
+    {% if chapter.video %}
+      <div class="row mb-3">
+        <div class="col-sm-12 col-md-10 offset-md-1 text-center">
+          <div class="embed-responsive embed-responsive-16by9">
+            <iframe src="{{ chapter.video }}" class="embed-responsive-item" frameborder="0" allowfullscreen></iframe>
+          </div>
         </div>
       </div>
-    </div>
+    {% endif %}
+    {% render_html_field chapter.introduction %}
+  {% else %}
+    {% with model=chapter %}
+      {% include "generic/not-available-warning.html" %}
+    {% endwith %}
   {% endif %}
-  {% render_html_field chapter.introduction %}
 
   <h2>Sections in this chapter:</h2>
-  <ol>
+  <ul class="list-unstyled">
     {% for section in chapter_sections %}
       <li>
-        <a href="{% url 'chapters:chapter_section' chapter.slug section.slug %}">
+        {{ chapter.number }}.{{ section.number }}
```

Comment:
For consistency with other parts of the website, the number always ends in a full stop.
```suggestion
        {{ chapter.number }}.{{ section.number }}.
```